scholly/base/src/global/data.js

```diff
@@ -12,17 +12,18 @@ import {
   FAKE_TEAM_MEMBERS,
   FAKE_TEAM_HOMEPAGE,
   FAKE_WEB_PAGES,
-  FAKE_TEAM_AWARDS,
+  FAKE_TEAM_ACHIEVEMENTS,
   FAKE_LAYOUT_OPTION,
 } from './fakeData';
 
 const env = process.env; // eslint-disable-line prefer-destructuring
+const defaultLayout = 1;
 let TEAM_PUBLICATIONS;
 let TEAM_INFO;
 let TEAM_MEMBERS;
 let TEAM_HOMEPAGE;
 let WEB_PAGES;
-let TEAM_AWARDS;
+let TEAM_ACHIEVEMENTS;
```

Comment:
You don't need to re-declare it since it's already a part of `mainline`: https://github.com/Researchify/Researchify/blob/main/scholly/base/src/global/data.js#L24